internal/pkg/controllers/profile/profile.go

```diff
@@ -120,32 +89,18 @@ type Reconciler struct {
 	save   saver
 }
 
-// Reconcile reconciles a SeccompProfile or a ConfigMap representing a seccomp profile.
+// Reconcile reconciles a SeccompProfile.
 func (r *Reconciler) Reconcile(req reconcile.Request) (reconcile.Result, error) {
 	logger := r.log.WithValues("profile", req.Name, "namespace", req.Namespace)
 
 	ctx, cancel := context.WithTimeout(context.Background(), reconcileTimeout)
 	defer cancel()
 
-	// Look for the SeccompProfile Kind first
-	seccompProfile := &seccompoperatorv1alpha1.SeccompProfile{}
-	if err := r.client.Get(ctx, req.NamespacedName, seccompProfile); err != nil {
-		logger.Error(err, "unable to fetch SeccompProfile")
-		seccompProfile = nil
-	}
-	// If no SeccompProfile, look for a ConfigMap
-	configMap := &corev1.ConfigMap{}
-	if seccompProfile == nil {
-		if err := r.client.Get(ctx, types.NamespacedName{Name: req.Name, Namespace: req.Namespace}, configMap); err != nil {
-			// Returning an error means we will be requeued implicitly.
-			return reconcile.Result{}, errors.Wrap(ignoreNotFound(err), errGetProfile)
-		}
-	}
-
 	// Pre-check if the node supports seccomp
 	if !seccomp.IsSupported() {
 		err := errors.New("profile not added")
 		logger.Error(err, fmt.Sprintf("node %q does not support seccomp", os.Getenv(config.NodeNameEnvKey)))
+		seccompProfile := &seccompoperatorv1alpha1.SeccompProfile{}
 		r.record.Event(seccompProfile,
```

Comment:
I see no need to bind the variable:
```suggestion
		r.record.Event(&seccompoperatorv1alpha1.SeccompProfile{},
```